docs/integrate/datagrip/index.md

```diff
@@ -0,0 +1,108 @@
+(datagrip)=
+# DataGrip
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://blog.jetbrains.com/wp-content/uploads/2019/01/datagrip_icon.svg){w=120px}](https://www.jetbrains.com/datagrip/)
+```
+
+[DataGrip] is a cross-platform database IDE that is tailored to suit the
+specific needs of professional SQL developers.
+
+It is available as a standalone application and is also included into
+other JetBrains products like IntelliJ IDEA and PyCharm.
+
+Connecting DataGrip to CrateDB uses the [CrateDB JDBC Driver].
+
+```{div}
+:style: "clear: both"
+```
+
+
+## Install
+
+::::{grid} 2
+:::{grid-item}
+For connecting to CrateDB, install the [CrateDB JDBC Driver]
+using the "Custom JARs" option when adding a database driver.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/a8c1ada6-fd97-43f4-a1ba-91aba1520bdb){h=180px}
+![Image](https://github.com/user-attachments/assets/1f925848-fac3-4265-8bd3-96f91daf03c9){h=180px}
+:::
+:::{grid-item}
+[crate-jdbc-standalone] is the right choice here.
+For example, download and use the [crate-jdbc-standalone-2.7.0.jar] JAR file,
```

Comment:
Do we really want to link to a specific version here?
There is already being work done to potentially upgrade this soon.

Reply:
Not necessarily, but a direct download link is also convenient.

Reply:
Couldn't we then at least have a central `[crate-jdbc-standalone-latest]` link and reuse this across examples?

Reply:
Good idea. Implemented per c270a6541b.